great_expectations/rule_based_profiler/parameter_builder/regex_pattern_string_parameter_builder.py

```diff
@@ -101,12 +101,7 @@ def threshold(self) -> Union[str, float]:
     @property
     def candidate_regexes(
         self,
-    ) -> Union[
-        str,
-        Union[
-            Set[str], List[str], "RegexPatternStringParameterBuilder.CANDIDATE_REGEX"
-        ],
-    ]:  # noqa: F821
+    ) -> Union[str, Union[List[str], Set[str],],]:
```

Comment:
A little confused by this type annotation - should the commas be here?

Reply:
@cdkini That was there before, I just got rid of the unnecessary (and incorrect) quoted type and simplified it.  I got rid of the commas (and `black` seems to be OK with it -- in next commit).